TwoJugWaterPuzzle/00_two_jug_water_puzzle.py

```diff
@@ -0,0 +1,56 @@
+"Implementation of the two jug water puzzle."
+
+
+def gcd(a, b):
+    "Return the greatest common divisor of a and b."
+    if b == 0:
+        return a
+    else:
+        return gcd(b, a % b)
+
+
+def pour(toJugCap, fromJugcap, d):
+    """
+    fromCap -- Capacity of jug from which
+              water is poured
+    toCap   -- Capacity of jug to which
+              water is poured
+     d       -- Amount to be measured
+    """
+    fromJug = fromJugcap
+    toJug = 0
```

Comment:
This is Python, not JavaScript.
```suggestion
    from_jug = from_jug_capacity
    to_jug = 0
```